adapter/PreciceInterface.c

```diff
@@ -330,13 +328,13 @@ void PreciceInterface_Create( PreciceInterface * interface, SimulationData * sim
 	interface->preciceNodeIDs = NULL;
 	interface->triangles = NULL;
 	interface->nodeScalarData = NULL;
-        interface->nodeVectorData = NULL;
+    interface->nodeVectorData = NULL;
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).